libs/gi/ui/src/util/getDisplayUtil.tsx

```diff
@@ -114,6 +115,7 @@ export function getDisplaySections(data: UIData): DisplaySections {
   )
   return [
     ...basic,
+    ['character', data.getCharacterStats()],
```

Comment:
Character stat comparison showing % increase seems a bit silly

Reply:
I wonder if it makes sense not to show % increase for values below 100?